pom.xml

```diff
@@ -6216,6 +6217,7 @@
           <fork>true</fork>
           <source>${maven.compiler.source}</source>
           <target>${maven.compiler.target}</target>
+          <release>${maven.compiler.release}</release>
```

Comment:
Are Java 8 builds fine with the presence of this setting?

Reply:
Fixed